addons/l10n_vn/models/res_bank.py

```diff
@@ -66,6 +73,15 @@ def _get_error_messages_for_qr(self, qr_method, debtor_partner, currency):
 
     def _check_for_qr_code_errors(self, qr_method, amount, currency, debtor_partner, free_communication, structured_communication):
         if qr_method == 'emv_qr' and self.country_code == 'VN' and self.proxy_type not in ['merchant_id', 'payment_service', 'atm_card', 'bank_acc']:
```

Comment:
```suggestion
        if qr_method != 'emv_qr' or self.country_code != 'VN':
```
Otherwise you will impact all the other QR methods

Reply:
Thanks, done